man/mustache.5.ron

```diff
@@ -411,6 +411,37 @@ Can be thought of as a single, expanded template:
     {{/names}}
 
 
+**Dynamic Names**
+
+Partials can be loaded dynamically at runtime using Dynamic Names; an
+**optional** special notation which allows to dynamically determine a tag's
+content at runtime.
+
+Dynamic Names consists of an asterisk, followed by a dotted name which follows
+the same notation and the same resolution as in an Interpolation tag. That is
+`{{>*dynamic}}`.
```

Comment:
Perhaps compare this to the hypothetical `{{>{{dynamic}}}}` to further clarify what it does.

Reply:
I think that including wrong examples may be confusing